### PR TITLE
Feature/v1

### PR DESCRIPTION
**Pull request by hixi**
first (working) version.

Roughly reviewed by:
- [x] @das-g

### CODE REVIEW

File: build_and_push_docker_images.sh [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+  docker build -f "${CONVERSION_SERVICE_LOCAL_REPO}/Dockerfile.rest_api" -t $SERVICE_IMAGE_NAME $CONVERSION_SERVICE_LOCAL_REPO
+  docker build -f "${CONVERSION_SERVICE_LOCAL_REPO}/Dockerfile.worker" -t $WORKER_IMAGE_NAME $CONVERSION_SERVICE_LOCAL_REPO
+  docker build -f "${WEBAPP_LOCAL_REPO}/Dockerfile" -t $WEBAPP_IMAGE_NAME $WEBAPP_LOCAL_REPO
+
+  docker push geometalab/osmaxx-conversion-worker:latest
+  docker push geometalab/osmaxx-conversion-service:latest
+  docker push geometalab/osmaxx-webapp:latest
+
+  rm -rf $CONVERSION_SERVICE_LOCAL_REPO
+  rm -rf $WEBAPP_LOCAL_REPO
```
Comment (das-g) [inline build_and_push_docker_images.sh]: Could the deletions be done before the `docker push`es? Then the repositories wouldn't stay laying around if the pushing fails.

File: README.md [hunk 2]
```diff
@@ -1,2 +1,43 @@
...
+* bootstrap (only the first time)
+* deploy
+
+Example:
+
+```bash
+git clone git@github.com:geometalab/osmaxx-deployment-scripts.git
+cp .env-dist .env
+vim .env
+python3 ./deployer.py
```
Comment (das-g) [inline README.md]: should `deployer.py` get a `#!/usr/bin/env python3` and made executable?

File: deployer/compose_commands.py [hunk 3]
```diff
@@ -0,0 +1,152 @@
...
+        assembles docker run with compose-image command with all options.
+
+        :param extra_environment_variables: environment variables that are only needed in this context (default: None).
+        :return: list of subprocess.check_call suitable strings
+        """
+        return self._get_docker_run_command + \
+               self._volumes +\
+            self._get_environment_variables(extra_environment_variables) +\
+            self._options +\
+            self._image
```
Comment (das-g) [inline deployer/compose_commands.py]: What's up with the indentation here? And shouldn't this be a `"".join((..., ))`?

Comment (hixi) [inline deployer/compose_commands.py]: indentation is messed up.

No, it should not be joined, these are lists.

**Pull request merged**